pkg/server/cloud/cloud.go

```diff
@@ -112,12 +112,14 @@ type SubnetAttributes struct {
 }
 
 type Image struct {
-	ID             string
-	Name           string
-	RootDevice     string
-	CreationTime   *time.Time
-	VolumeDiskSize int32 // in GiB
-	VolumeType 	   string
+	ID               string
+	Name             string
+	RootDevice       string
+	CreationTime     *time.Time
+	VolumeDiskSize   int32 // in GiB
+	VolumeType       string
+	VolumeIops       int64
+	VolumeThroughput int64
```

Comment:
Same here for `VolumeIops` and `VolumeThroughput`, it's better to make them pointers.

Reply:
changed `EbsSpec` & `cloud.Image` Throughput & Iops for pointers :+1: